src/Samples/Toolkit.SampleApp.Maui/Samples/PopupViewerSample.xaml.cs

```diff
@@ -121,5 +104,26 @@ private Popup GetPopup(IEnumerable<IdentifyLayerResult> results)
 
             return null;
         }
+
+        private void OnBackgroundTapped(object sender, EventArgs e)
+        {
+            popupPanel.IsVisible = false;
+        }
+
+        private void CloseButton_Click(object sender, EventArgs e)
+        {
+            popupPanel.IsVisible = false;
+        }
+
+        private void popupViewer_PopupAttachmentClicked(object sender, Esri.ArcGISRuntime.Toolkit.Maui.PopupAttachmentClickedEventArgs e)
```

Comment:
Are these handlers supposed to be hooked up? They look unused now.

Reply:
OnBackgroundTapped isn't used (will remove). The other two are hooked up in the xaml